src/test/java/com/wikia/webdriver/testcases/signuptests/SignUpTests.java

```diff
@@ -222,4 +224,21 @@ public void SignUp_008_signupJapaneseUser() {
     preferences.selectTab(tabNames.EMAIL);
     preferences.verifyEmailMeSection();
   }
+
+  @BeforeMethod(alwaysRun = true)
+  public void setEnvironmentCredentials() {
```

Comment:
I disagree, since we need to run the tests with different users based on environment only for these two specific instances.  I think this is the right way to do it - @ludwikkazmierczak what do you think?